Python/pylifecycle.c

```diff
@@ -648,9 +648,11 @@ pycore_init_builtins(PyThreadState *tstate)
     if (interp->builtins_copy == NULL) {
         goto error;
     }
+    Py_DECREF(bimod);
     return _PyStatus_OK();
 
 error:
+    Py_DECREF(bimod);
```

Comment:
It should be Py_XDECREF for the if (bimod == NULL) { goto error; } case.

Reply:
Will update soon

Reply:
@vstinner Updated in d00de0e